pkgs/development/python-modules/soundcard/default.nix

```
@@ -0,0 +1,43 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, libpulseaudio
+, python3Packages
```

the python-modules dir will parameterize the python packages, do not import a specific python package set
```suggestion
, numpy
, cffi
```